src/cgtop/cgtop.c

```diff
@@ -207,7 +207,7 @@ static int process(
                         return r;
 
                 g->n_tasks = 0;
-                while (cg_read_pid(f, &pid) > 0) {
+                while (cg_read_pid(f, &pid, /* flags = */ 0) > 0) {
```

Comment:
hmm, i think here we should actually count the zero tasks too, i.e. n_tasks whould be increased below if we see a zero pid

Reply:
Yeah, just pushed a new version with that changed.

Reply:
Hmm, this is changed back? It should still set `CGROUP_INCLUDE_UNMAPPABLE_PID`?